_src/css/_body.scss

```diff
@@ -17,96 +17,50 @@
     box-shadow: 3px 2px 10px 0 white;
     margin: 25px;
     display: flex;
-    flex-direction: column;
+    justify-content: space-around;
     border-radius: 2%;
   }
 
-  &__info{
-    display:flex;
-    height: 90%;
-  }
-
   &__frame {
     width: 30%;
     display: flex;
     flex-direction: column;
     position: relative;
     justify-content: center;
     align-items: center;
+    bottom: 15px;
```

Comment:
This rule shouldn't be here. It is pushing the entire component out of its parent